engine/src/main/java/org/hibernate/search/spatial/impl/SpatialIndexReader.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.spatial.impl;
+
+import java.io.IOException;
+
+import org.apache.lucene.index.AtomicReader;
+import org.apache.lucene.index.FieldInfo;
+import org.apache.lucene.index.NumericDocValues;
+import org.hibernate.search.util.logging.impl.Log;
+import org.hibernate.search.util.logging.impl.LoggerFactory;
+
+/**
+ * Utility class containing functions to read the value of a spatial field from an index.
+ *
+ * @author Davide D'Alto
+ */
+public final class SpatialIndexReader {
```

Comment:
We usually use `XYZHelper` to indicate helper classes with static methods. So I'd prefer `SpatialIndexReaderHelper` or just `SpatialHelper`. This avoids also the initial thought that we are dealing with a subclass of Lucene's `IndexReader` which was my first thought.